tools/simulator/frameworks/runtime-src/proj.ios_mac/mac/SimulatorApp.mm

```diff
@@ -383,7 +383,9 @@ - (void) createWindowAndView
     _window.contentView = viewController.view;
     [_window.contentView setWantsBestResolutionOpenGLSurface:YES];
     [_window makeKeyAndOrderFront:nil];
-
+    
+    NSApplication *thisApp = [NSApplication sharedApplication];
+    [thisApp activateIgnoringOtherApps:YES];
```

Comment:
Why need it? What does it do?

Reply:
the App Windows on MacOS wouldn't  show  up on the front without  calling  this  API

Reply:
see this issue: https://github.com/cocos-creator/3d-tasks/issues/5437